.github/workflows/publish-to-test-pypi.yml

```diff
@@ -0,0 +1,90 @@
+name: Publish Python 🐍 distribution 📦 to PyPI
+
+on: push
+
+jobs:
+  build:
+    name: Build distribution 📦
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: "3.10.12"
+    - name: Install specific setuptools and pip
+      run: |
+        python -m pip install setuptools==65.5.0 pip==21
+    - name: Install pypa/build
+      run: |
+        python3 -m pip install build --user
```

Comment:
Why does this change from python to python3? Seems inconsistent

Reply:
I agree, let's make it `python3`